internal/operatorcli/operator_cli.go

```diff
@@ -550,20 +585,19 @@ func RunEOChainSetAlias(c *cli.Context) error {
 			crypto.PubkeyToAddress(ecdsaPair.PublicKey),
 		)
 		if err != nil {
-			return cli.Exit(fmt.Sprintf("Failed to create EOConfig.SetAlias transaction %v", err), 1)
+			return cli.Exit(fmt.Sprintf("Failed to create EOConfig.declareAlias transaction %v", err), 1)
 		}
 
 		ctx := context.Background()
 		receipt, err := txMgr.Send(ctx, tx)
 		if err != nil {
-			return cli.Exit(fmt.Sprintf("setAlias transaction failed %s", err), 1)
+			return cli.Exit(fmt.Sprintf("declareAlias transaction failed %s", err), 1)
 		}
 		if receipt.Status != 1 {
-			return cli.Exit(fmt.Sprintf("setAlias transaction %v reverted", receipt.TxHash.Hex()), 1)
+			return cli.Exit(fmt.Sprintf("declareAlias transaction %v reverted", receipt.TxHash.Hex()), 1)
 		}
 
-		logger.Info("succesfully set the alias in the eochain", "Ethereum address", signerAddr, "eochain address", crypto.PubkeyToAddress(ecdsaPair.PublicKey), "tx hash", receipt.TxHash.Hex())
-
+		logger.Info("succesfully declare the alias in the eochain", "Ethereum address", signerAddr, "eochain address", crypto.PubkeyToAddress(EthEcdsaPair.PublicKey), "tx hash", receipt.TxHash.Hex())
```

Comment:
1. Successfully declared an alias address on the EOChain
2.  the addresses in the this log line are the same no? second one should be the new alias